aip/general/0122.md

```diff
@@ -294,6 +294,10 @@ equivalent to the corresponding message's name in snake case.
 - Fields representing another resource **should** provide the
   `google.api.resource_reference` annotation with the resource type being
   referenced.
+- Services **should** use the
+resource name to reference resources when possible. If using the ID component
```

Comment:
I don't think the first sentence is necessary. This guidance is mentioned the item immediately before. This item, imo, should be just about when an ID component is used.

Reply:
sounds good. If so I'll probably try to refine the original item above. 

I think this is the guidance you're referring to that says we should use the resource name format?

```
When referencing a resource name for a different resource, the field **should**
be of type `string` for the resource name, and the field name **should** be
equivalent to the corresponding message's name in snake case.
```

Reply:
I removed the redundant sentence as advised, and clarified some wording. PTAL.